### DISCUSSION TITLE
uso de requisições https

### DISCUSSION THREAD

**Discussion by hsswar** [viniciussanchez/RESTRequest4Delphi#215] [General]
Olá uso Delphi xe7, se eu usar  RestRequest4D para requisições https me dá erro , ja em Delphi xe 11 por exemplo, esse tipo de erro não tenho, sabe me dizer se tem alguma restrição ou atualização para isso?

**Comment by viniciussanchez** [comment]
Provavelmente será necessário alguns ajustes por meio de diretivas de compilação no RR4D para funcionar no XE7.

**Reply by sandroreck** [reply]
Tenho o mesmo problema, quais seriam as indicações de diretivas de compilação? Alguma dica?

Aqui exemplo, funciona no XE11 e no XE7 não:

     Resp := TRequest.New.BaseURL(_URL_BASE + _URL_TOKEN_GET)
             .AddHeader('Ocp-Apim-Subscription-Key', _TOKEN_APP)
             .ContentType('application/json')
             .Accept('*/*')
             .AcceptEncoding('gzip, deflate, br')
             .TokenBearer(_TOKEN_AUT)
             .AddBody(jsonObj)
             .Post;
